charts/leonardo/templates/_configMap.tpl

```diff
@@ -0,0 +1,27 @@
+{{- /* Generate the data component of a Cromwell config map */ -}}
```

Comment:
these are not used yet...But probably will be needed for Leonardo, hence leaving it there for now